mautic/mautic
# Add a tab with devices

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | enhancement
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This simply adds a devices tab to the details tray of a contact profile.

![](http://alan.direct//drop/2019-01-15_18-29-25.386.png)
[//]: # ( As applicable: )

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2.   Ensure that the contact has devices then go to their profile (/s/contacts/view/ID), click the [Details] tray then the Devices tab. Notice the info.
